mongodbatlas/resource_mongodbatlas_encryption_at_rest.go

```diff
@@ -239,27 +240,41 @@ func resourceMongoDBAtlasEncryptionAtRestCreate(ctx context.Context, d *schema.R
 		encryptionAtRestReq.GoogleCloudKms = expandGCPKmsConfig(gcpC.([]interface{}))
 	}
 
-	for i := 0; i < 5; i++ {
-		_, _, err := conn.EncryptionsAtRest.Create(ctx, encryptionAtRestReq)
+	timeout := d.Timeout(schema.TimeoutCreate)
+	stateConf := &retry.StateChangeConf{
+		Pending:    []string{"RETRY"},
+		Target:     []string{"COMPLETED", "ERROR"},
```

Comment:
[nit] Should we make RETRY/COMPLETE/ERROR constants and use them in all the resources with a similar strategy?